e107_admin/admin.php

```diff
@@ -229,7 +229,8 @@ private function checkPaths()
 				}
 				else
 				{
-					$mes->addWarning("Unable to create <b>".$dr."</b>. Please check your folder permissions.");
+					$message = e107::getParser()->lanVars(e107::getParser()->toHTML(ADLAN_187, true), array('x'=>$dr));
```

Comment:
Instead of using `toHtml()` use `lanVars(LAN,$dr,true);` to make the variable bold. There is no need to use `array('x'=>$dr)` when only one variable of [x] is in the LAN.